gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadChannel.java

```diff
@@ -179,6 +180,26 @@ public GoogleCloudStorageReadChannel(
     }
   }
 
+  public GoogleCloudStorageReadChannel(
```

Comment:
Do we need this constructor to be public ?

Reply:
No. Will change it back.